templates/repo/issue/view_content/pull.tmpl

```diff
@@ -49,7 +49,12 @@
 		<div class="ui {{if not $.LatestCommitStatus}}top attached header{{else}}attached merge-section segment{{end}}">
 			{{if .Issue.PullRequest.HasMerged}}
 				<div class="item text purple">
-					{{$.i18n.Tr "repo.pulls.has_merged"}}
+					{{if .Issue.PullRequest.MergedCommitID}}
+						{{$link := printf "%s/commit/%s" $.Repository.HTMLURL .Issue.PullRequest.MergedCommitID}}
+						{{$.i18n.Tr "repo.pulls.merged_as" $link (ShortSha .Issue.PullRequest.MergedCommitID) | Safe}}
+					{{else}}
```

Comment:
Just a question. When will the `else` happen? Or is this just a defensive programming practice?

Reply:
I think a very old version of gitea will have that column empty since many users begined gitea from 1.0 .

Reply:
Okay, got it.